src/client.ts

```diff
@@ -115,6 +123,21 @@ export class FarosClient {
     }
   }
 
+  async addCanonicalModels(graph: string): Promise<void> {
```

Comment:
change to `async addModels(graph: string, schema = 'canonical', models?: string)`